resources/install.rb

```diff
@@ -0,0 +1,53 @@
+# resources/install.rb
+
+# Resource for InfluxDB install
+
+property :influxdb_key, String, default: 'https://repos.influxdata.com/influxdb.key'
+property :arch_type, String, default: 'amd64'
+default_action :install
+
```

Comment:
I would recommend allowing the addition of these repos to be optional, like is done here:

https://github.com/NorthPage/telegraf-cookbook/blob/master/resources/install.rb